src/routes/Settings.tsx

```diff
@@ -99,12 +98,15 @@ const Settings: React.FC = () => {
       </>
     );
   };
-
   return (
     <div className={classes.container}>
       <Title>Deine Informationen</Title>
       <SettingsCard user={userContext.user} />
-      {!isProjectCoachButNotTutor(userContext.user) && renderSubjects()}
+      {!userContext.user.isTutor &&
```

Comment:
If the user is tutor, he SHOULD see the subjects.

Reply:
@lsc0498 i tried to explain it in a comment above
but here it is again

-> isTutor is set to false when tutor is displayed on the side sreen in thapp(so for now we wrote the code to function with what we have
on the screen)
bec. i don't know exactly if the problem is only in the frontend or is there also a problem in the backend

Reply:
What do you mean by side screen?

Reply:
in the part where you see the settings and 1 on 1 mentoring... meant the side menu

Reply:
![Capture](https://user-images.githubusercontent.com/40523755/103179669-9c505b80-488e-11eb-9bcd-d5fbcb258828.PNG)
![Capture2](https://user-images.githubusercontent.com/40523755/103179671-9ce8f200-488e-11eb-9441-6752896ac75e.PNG)

Reply:
There it should only be indicated whether the user is "Student*in" or "Schüler*in" which actually should have nothing to do with `isTutor`.

Reply:
@lsc0498 examples of the problem above.. here i console.log the value for the isTutor boolean variable, and here ou can see that when isTutor is true -> it is not shown as a tag and vice versa

Reply:
I think the problem is that you haven't pulled before making this branch ;) Please merge `master` into your branch.

Reply:
ooh :) , thx .. will try to fiy the rest as soon as possible